### ISSUE TITLE
`n_folds` instead of `nfolds` is passed to `hal9001` and therefore to `glmnet`

### ISSUE THREAD

**Issue by jeremyrcoyle** [nhejazi/haldensify#41]
`Lrnr_haldensify` in `sl3` is generating the following warning: 

```
Warning message:
In (function (X, Y, formula = NULL, X_unpenalized = NULL, max_degree = ifelse(ncol(X) >=  :
  Some fit_control arguments are neither default nor glmnet/cv.glmnet arguments: n_folds; 
They will be removed from fit_control
```

**Comment by nhejazi** [comment]
resolved by #42, to be closed after merging that PR

**Comment by nhejazi** [comment]
resolved by [58822ff](https://github.com/nhejazi/haldensify/commit/58822ff44fc04a6e286c0e241a5f613a0bdc759a)
